lib/Checks/AlwaysUseBracesCheck.vala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2018 elementary LLC. (https://github.com/elementary/vala-lint)
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program; if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ */
+
+public class ValaLint.Checks.AlwaysUseBracesCheck : Check {
+    public AlwaysUseBracesCheck () {
+        Object (
+            title: _("always-use-braces"),
+            description: _("Checks for braces")
+        );
+    }
+
+    public override void check (Gee.ArrayList<ParseResult?> parse_result, ref Gee.ArrayList<FormatMistake?> mistake_list) {
+        foreach (ParseResult r in parse_result) {
+            if (r.type == ParseType.DEFAULT) {
+                add_regex_mistake ("""\)\s*\n""", _("Expected an opening brace"), r, ref mistake_list, 1);
```

Comment:
I think this gives too many false positives when tested on some real elementary code.  I suggest you modify it so that it is only a mistake if the line contains a preceding `if (`